### DISCUSSION TITLE
Error while updating Paperless

### DISCUSSION THREAD

**Discussion by Spezialtrick** [tteck/Proxmox#1378] [Help Needed] [answered]
I've just tried to update Paperless with

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)"`

and getting the following error:

`[ERROR] in line 59: exit code 1: while executing command RELEASE=$(curl -s https://api.github.com/repos/paperless-ngx/paperless-ngx/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')`

**Comment by tteck** [comment, marked as answer]
Typically, the error you've provided suggests the presence of DNS problems, or github api issues. It's not a script issue.

**Reply by Spezialtrick** [reply]
It seems to be a GitHub issues. Updating is working now, without changes on my systems. Thanks!

**Comment by tteck** [comment]
You can check by running this command (from the script)
```
curl -s https://api.github.com/repos/paperless-ngx/paperless-ngx/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }'
```
you should receive `v1.14.2`
